world_map:
- [ 0.0,  0.0,   0, 0.2820, 0]
- [ 0.0, -1.0,   0, 0.2654, 1]
- [ 0.0, -2.0,   0, 0.2679, 2]
- [ 0.0, -3.0,   0, 0.2739, 3]
- [ 0.0, -4.0, -90, 0.1133, 4]
- [-1.0, -4.0, -90, 0.1207, 5]
- [-2.0, -4.0, -90, 0.2455, 6]
- [-3.0, -4.0, -90, 0.2748, 7]
- [-4.0, -4.0, -180, 0.2421, 8]
- [-4.0, -3.0, -180, 0.2281, 9]
- [-4.0, -2.0, -180, 0.2572, 10]
- [-4.0, -1.0, -180, 0.2230, 11]
- [-4.0, -0.0, -180, 0.2249, 12]
- [-3.0, -0.0,   90, 0.1695, 13]
- [-2.0, -0.0,   90, 0.2418, 14]
- [-1.0, -0.0,   90, 0.2452, 15]
